Fails param_eval>: when using custom variables in nested variables

in some situation, it gets fails. Could you please take a look?
```
$ digdag run param_eval.dig  --session hourly --rerun
2021-04-20 01:23:24 +0900: Digdag v0.9.42
2021-04-20 01:23:25 +0900 [INFO] (main): Using session /......../test_param_eval/.digdag/status/20210419T160000+0000.
2021-04-20 01:23:25 +0900 [INFO] (main): Starting a new session project id=1 workflow name=param_eval session_time=2021-04-19T16:00:00+00:00
2021-04-20 01:23:26 +0900 [INFO] (0018@[0:default]+param_eval+eval): param_eval>: test_message.title
2021-04-20 01:23:26 +0900 [ERROR] (task-thread-0): Uncaught exception. Task queue will detect this failure and this task will be retried later.
io.digdag.spi.TemplateException: Failed to evaluate a variable {"timezone":"UTC","session_uuid":"16c8da0f-7f32-41d2-949c-dca9f3ec7211","session_time":"2021-04-19T16:00:00+00:00","session_id":1,"session_date":"2021-04-19","session_date_compact":"20210419","session_local_time":"2021-04-19 16:00:00","session_tz_offset":"+0000","session_unixtime":1618848000,"last_executed_session_time":"","last_executed_session_date":"","last_executed_session_date_compact":"","last_executed_session_local_time":"","last_executed_session_tz_offset":"+0000","last_executed_session_unixtime":"","project_id":1,"attempt_id":1,"task_name":"+param_eval+eval","yesterday_ds":"2021-04-18","test_message":{"title":"yesterday is ${moment(session_date).add(-1, 'days').format(\"YYYY-MM-DD\")}"},"today_unixtime":"1618790400","plugin":{"repositories":["https://jitpack.io"],"dependencies":["pro.civitaspo:digdag-operator-param:0.0.2"]},"param_eval>":"test_message.title","_type":"param_eval","_command":"test_message.title","repositories":["https://jitpack.io"],"dependencies":["pro.civitaspo:digdag-operator-param:0.0.2"]} (SyntaxError: <function>:4:126 Missing close quote
((__t=(moment(session_date).add(-1, 'days').format(\"YYYY-MM-DD\")))==null?'':(typeof __t=="string"?__t:JSON.stringify(__t)))+
                                                                                                                              ^)
```

config

```
_export:
  yesterday_ds: ${moment(session_date).add(-1, 'days').format("YYYY-MM-DD")}
  test_message: 
    title: yesterday is ${yesterday_ds}
  plugin:
    repositories:
      - https://jitpack.io
    dependencies:
      - pro.civitaspo:digdag-operator-param:0.0.2

+eval:
  param_eval>: test_message.title

+test:
  echo>: ${test_message.title}
```

purpose

without using this plugin, we cannot extract ${...} variable to plain text for sh operater.
I wold like to generate parameter injected sql and then pass to the embulk without pass all of parameter for liquid template. because I'm calling embulk with sh> operator

--- a/README.md
+++ b/README.md
@@ -102,7 +102,10 @@ _export:
 ### Options
 
 - **param_eval>**: Param name to eval. (string, required)
-  - **NOTE**: This operator is a workaround for the issue: [Exported vars are not evaluated recursively in the context of nested params](https://github.com/treasure-data/digdag/issues/862)
+
+**NOTE**: 
+* This operator is a workaround for the issue: [Exported vars are not evaluated recursively in the context of nested params](https://github.com/treasure-data/digdag/issues/862)
+* Use single quote`'` instead of double quote`"` for injecting javascript code to avoide `SyntaxError: <function>: Missing close quote` error: [Fails param_eval>: when using custom variables in nested variables](https://github.com/civitaspo/digdag-operator-param/issues/14)
 
 # Development
 

